### PR TITLE
Read a Book

### PR DESCRIPTION
**Pull request by Veegas**

### CODE REVIEW

**Comment by Veegas** [comment]
@Mernawaseif  
@MinaNemr
 @nelsammak
 @MinaNemr 
Review ya welad

File: frontend/components/epubjs/reader.js [hunk 1]
```diff
@@ -0,0 +1,1152 @@
+EPUBJS.reader = {};
+EPUBJS.reader.plugins = {}; //-- Attach extra Controllers as plugins (like search?)
+
```
Comment (MinaNemr) [inline frontend/components/epubjs/reader.js:3]: @Veegas name function document it

File: frontend/components/epubjs/reader.js [hunk 2]
```diff
@@ -0,0 +1,1152 @@
...
+	//exports to multiple environments
+	if (typeof define === 'function' && define.amd)
+	//AMD
+	define(function(){ return Reader; });
+	else if (typeof module != "undefined" && module.exports)
+	//Node
+	module.exports = ePubReader;
+
+})(window, jQuery);
+
```
Comment (MinaNemr) [inline frontend/components/epubjs/reader.js:28]: @Veegas document

File: frontend/components/epubjs/reader.js [hunk 3]
```diff
@@ -0,0 +1,1152 @@
...
+			len = bookmarks.length,
+			i;
+	
+	for(i = 0; i < len; i++) {
+		if (bookmarks[i]['cfi'] === cfi) return i;
+	}
+	return -1;
+};
+*/
+
```
Comment (MinaNemr) [inline frontend/components/epubjs/reader.js:211]: functions needs to be documented

File: frontend/scripts/controllers/reader.js [hunk 4]
```diff
@@ -1,12 +1,83 @@
...
+        * @function renderBook
+        * Renders the book
+        */
+        $scope.$on('$viewContentLoaded', function(event) {
+            $timeout(function renderBook() {
+                var are = $('#area')[0];
+
+                console.log(are);
+
+
```
Comment (MinaNemr) [inline frontend/scripts/controllers/reader.js:25]: need docuemnation

**Comment by MinaNemr** [comment]
@Veegas some functions aren't documented  
but i do suggest that we skip them for now

**Pull request merged**